src/StandardError.js

```diff
@@ -6,25 +6,25 @@ const StandardError = {
 	// Output the StandardError Object, optionally filtered down to a particular domain
 	show(domain) {
 		var map = {};
-		this.keys.map(key => this[key] &&
-			                 (!domain || this[key].domain == domain)
-			                  ? map[key] = this[key]
+		this.keys.map(key => this[key] && typeof this[key] == 'function' &&
```

Comment:
As you mentioned, I think this check isn't doing anything since everything on this Object is going to be a function. If something was registered as a key but is _not_ a function then that's actually a problem that we should throw an error for. As is, this change would hide that error by returning `null` (in the sole case of `show`, but still). I think we pull this out.